sdmetrics/single_table/privacy/dcr_utils.py

```diff
@@ -0,0 +1,162 @@
+"""Distance to closest record measurement functions."""
+
+import pandas as pd
+
+from sdmetrics.utils import is_datetime
+
+
+def _calculate_dcr_value(s_value, d_value, col_name, metadata, range=None):
+    """Calculate the Distance to Closest Record between two different values.
+
+    Arguments:
+        s_value (int, float, datetime, boolean, string, or None):
+            The synthetic value that we are calculating DCR value for
+        d_value (int, float, datetime, boolean, string, or None):
+            The data value that we are referencing for measuring DCR.
+        col_name (string):
+            The column name in the metadata dictionary that will be used to
+            determine how the type is measured.
+        metadata (dict):
+            The metadata dict.
```

Comment:
Could we just pass in the column sdtype instead of the column name and entire metadata?